lib/beautify-stack.js

```diff
@@ -54,19 +59,20 @@ function extractFrames(stack) {
  * /home/ava/ex.js:12:5
  * outer (/home/ava/ex.js:13:4)
  * Object.<anonymous> (/home/ava/ex.js:14:3)
+ * Module._compile (module.js:570:32)
+ * Object.Module._extensions..js (module.js:579:10)
+ * Module.load (module.js:487:32)
+ * tryModuleLoad (module.js:446:12)
+ * Function.Module._load (module.js:438:3)
+ * Module.runMain (module.js:604:10)
  * ```
  */
 module.exports = stack => {
 	if (!stack) {
 		return '';
 	}
 
-	stack = extractFrames(stack);
-	// Workaround for https://github.com/tapjs/stack-utils/issues/14
-	// TODO: fix it in `stack-utils`
-	stack = cleanStack(stack);
-
-	return stackUtils.clean(stack)
+	return stackUtils.clean(extractFrames(stack))
```

Comment:
Do we still need `extractFrames`? Hopefully `stack-utils` deals with this now.

Reply:
Thinking about this more, do we even need to clean the stack here? We could do it in the reporters instead. We'd have to update `serialize-error` so it finds the first call site that is in the test file, rather than assuming it's the first call site in the trace.

Reply:
@novemberborn 
Do you think the parameter `shouldBeautifyStack` in function `serialize-error` (https://github.com/avajs/ava/blob/ede4f322b4fa6263c8ae14ec04282ab06bb0afd4/lib/serialize-error.js#L136) is still necessary? It's set to false for internal-errors in the modules `lib/subprocess/worker`(https://github.com/avajs/ava/blob/ede4f322b4fa6263c8ae14ec04282ab06bb0afd4/lib/worker/subprocess.js#L161) and `lib/api` (https://github.com/avajs/ava/blob/ede4f322b4fa6263c8ae14ec04282ab06bb0afd4/lib/api.js#L239).

Without this parameter, it would be much easier to move the stack-cleaning to the reporters.

Reply:
Yea that's what I'm saying. Let's see if we can move the stack processing to the main process (reporters).